prometheus-ksonnet/lib/config.libsonnet

```diff
@@ -87,13 +87,5 @@
 
     // Node exporter options.
     node_exporter_mount_root: true,
-
-    // oauth2-proxy
-    oauth_enabled: false,
-
-    // Nginx proxy_read_timeout (in seconds) 60s is the nginx default
-    nginx_proxy_read_timeout: '60',
-    // Nginx proxy_send_timeout (in seconds) 60s is the nginx default
-    nginx_proxy_send_timeout: '60',
```

Comment:
nit: It is not necessary to remove this block.

Reply:
fair enough, reverting this chagne.